lib/stimulus_reflex/configuration.rb

```diff
@@ -14,11 +14,12 @@ def configuration
   end
 
   class Configuration
-    attr_accessor :on_failed_sanity_checks, :parent_channel
+    attr_accessor :on_failed_sanity_checks, :parent_channel, :logging
 
     def initialize
       @on_failed_sanity_checks = :exit
       @parent_channel = "ApplicationCable::Channel"
+      @logging = []
```

Comment:
@piotrwodz Should we maybe provide a default logging? People upgrading will not have the initializer with your new configuration.

Reply:
100%

It's likely that once we have the syntax nailed down, we'll copy what's in the template so it's the actual default; reinforcing the educational potential. It likely will use fewer colors 😀

Reply:
Definitely.